app/views/links/show.html.erb

```diff
@@ -166,6 +166,18 @@
 <%= turbo_stream_from "link_details_#{@link.id}" %>
 <%= render 'details', link: @link, set_by: @set_by %>
 
+<% unless @past_links.empty? %>
+  <h3>Previous wallpapers</h3>
+  <%= render partial: 'past_links', locals: { past_links: @past_links } %>
+  <a href="<%=link_past_links_path(link_id: @link.id)%>" class="small see-all-past-wallpapers" %>
+    View More 
+    <% if current_user == @link.user %>
+      and Send Reactions
+    <% end %>
+    <ion-icon role="presentation" name="arrow-forward"></ion-icon>
+  </a>
+<% end %>
+
```

Comment:
Cool! So my only thing, and maybe it's worth asking some folks in the discord, but I think this moves the form to actually set the wallpaper down even further.

It's already a little crowded, so I've tried to keep the anon experience at least clean. When someone just arrives at the site, after receiving someone's link, they shouldn't really have to contend with a lot of "walltaker" things.

Maybe if we shrink the past link carousel down a bit or rearrange the page?

Reply:
That's fair, this does take up a lot of space. I'll think about it a bit and let you know if I come up with anything.

Reply:
Moving it to the bottom of the page for now. There might be some good opportunities for a reorganization/redesign of the page, but i'm going to call it "out of scope" in order to mask my laziness